### PR TITLE
Newcomers_Guide.rst: Gitter login using GitHub

### PR DESCRIPTION
**Pull request by palash25**
Suggestion for newcomers to join the gitter community using their GitHub account has been added to Newcomers_Guide.rst

Fixes: [#4466](https://github.com/coala/coala/issues/4466) 

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by NiklasMM** [comment]
As you see, gitmate has found some issues with your commit. You can recreate them locally, by running `coala` over your repository. Please fix them and amend them to your commit with `git commit --amend´

**Comment by Asnelchristian** [comment]
@palash25 please have a look at http://coala.io/commit, modify your commit message to follow the guidelines and we are good :) :+1:

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -113,6 +113,10 @@ The list of all available chatrooms are available here - `channel list <https://
 But before joining the community, here are few things that you should
 keep in mind.
 
+  * Only log into Gitter using your GitHub account and not your Twitter account
+    since the Gitter bot corobo identifies each user from their GitHub username
```
Comment (Asnelchristian) [inline docs/Developers/Newcomers_Guide.rst]: enclose '''corobo'''  within backsticks "``"

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -113,6 +113,10 @@ The list of all available chatrooms are available here - `channel list <https://
...
+    which makes it possible to automate certain tasks such as asking the bot to
+    assign an issue to your profile.
```
Comment (Asnelchristian) [inline docs/Developers/Newcomers_Guide.rst]: add some few more stuffs to this list ;) (let's show them how awesome corobo is)

Comment (NiklasMM) [inline docs/Developers/Newcomers_Guide.rst]: I disagree, this is outside the scope of the issue being worked on here and the corobo intro is at another point in the file.

**Comment by palash25** [comment]
@Asnelchristian enclosed '''corobo''' within backticks and edited the commit message according to the guidelines. Let me know if any more changes are required.

**Comment by yash-nisar** [comment]
As mentioned [here](https://github.com/coala/coala/pull/4476#pullrequestreview-50656046) by gitmate your line is of 81 chars (above 80). Please fix that. :smile:

**Comment by yash-nisar** [comment]
You can run coala locally to get rid of gitmate issues :+1:

**Comment by Asnelchristian** [comment]
@palash25 just rebase and :tada: :+1:

**Comment by palash25** [comment]
@Asnelchristian how do I rebase?

**Comment by yash-nisar** [comment]
See http://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing

**Comment by palash25** [comment]
Done. Rebased, thanks Yash Nisar

**Comment by yash-nisar** [comment]
You haven't rebased. :confused:

**Comment by palash25** [comment]
Yeah you're right I just checked. Do I have to push it to my fork after rebasing.

**Comment by Asnelchristian** [comment]
git push --force

**Comment by palash25** [comment]
@yash-nisar  when I fetched and rebased I was in my development branch but I didn't push anything to my fork after that and I closed git bash. Now my fork's master branch is 2 commits behind the original and the development branch is one commit ahead and 2 commits behind the original.

**Comment by palash25** [comment]
I did git push --force but now my fork says that the master is still 2 commits behind the original and the development branch just 1 commit behind. Do I need to rebase and push all over again?

**Comment by NiklasMM** [comment]
Did you make sure to fetch the current state of the upstream master and then rebase on the upstream master?

**Comment by palash25** [comment]
This is what I did 
    $ git fetch upstream
    $ git rebase upstream/master
but then I forgot to push it and closed git bash. After that I opened git bash again and pushed it to my fork but it still says it behind.

**Comment by yash-nisar** [comment]
@palash25 Come on coala.io/chat, we can help you out there :smile:

**Comment by NiklasMM** [comment]
ack b724a12

**Comment by sils** [comment]
ack b724a12

**Comment by sils** [comment]
@rultor merge
